site/content/_future.txt

```diff
@@ -15,6 +15,7 @@
       <a href="/events/2015-amsterdam">Amsterdam - June 2015</a><br/>
       <a href="/events/2015-singapore/">Singapore - 2015</a><br/>
       <a href="/events/2015-minneapolis/">Minneapolis - July 2015</a><br/>
+      <a href="/events/2015-minneapolis/">Melbourne - July 2015</a><br/>
```

Comment:
<strike>a href="/events/2015-minneapolis/">Melbourne - July 2015</strike>

Should be:

a href="/events/2015-melbourne/">Melbourne - July 2015<br/>